packages/fiori/src/Wizard.ts

```diff
@@ -251,7 +176,100 @@ const metadata = {
  * @appenddocs sap.ui.webc.fiori.WizardStep
  * @public
  */
+@customElement("ui5-wizard")
+@languageAware
+@fastNavigation
+
+/**
+ * Fired when the step is changed by user interaction - either with scrolling,
+ * or by clicking on the steps within the component header.
+ *
+ * @event sap.ui.webc.fiori.Wizard#step-change
+ * @param {WizardStep} step The new step.
+ * @param {WizardStep} previousStep The previous step.
+ * @param {boolean} changeWithClick The step change occurs due to user's click or 'Enter'/'Space' key press on step within the navigation.
+ * @public
+ */
+@event("step-change", {
+	detail: {
+		step: { type: WizardStep },
+		previousStep: { type: WizardStep },
+		changeWithClick: { Boolean },
+	},
+})
+
 class Wizard extends UI5Element {
+	/**
+	 * Defines the width of the <code>ui5-wizard</code>.
+	 * @private
+	 */
+	@property({ validator: Float })
```

Comment:
validators Integer, Float (or previously type Integer, FLoat) are null by default, 
and !: declares it as not-null value

Before assigned, `width` will be null